src/azureStorageHelper.ts

```diff
@@ -0,0 +1,69 @@
+'use strict';
+
+import * as storage from 'azure-storage';
+import { FileService } from 'azure-storage';
+import * as path from 'path';
+
+const DIRECTORY_NAME = 'ansible-playbooks';
+
+export async function uploadFilesToAzureStorage(localFileName: string, storageAccountName: string, storageAccountKey: string, fileShareName: string): Promise<void> {
+    const client = storage.createFileService(storageAccountName, storageAccountKey);
+
+    try {
+        await createFileShare(client, fileShareName);
+    } catch (err) {
+        console.log('Failed to create FileShare ' + fileShareName + ', detail ' + err);
```

Comment:
i am just a bit concerned that there's "console.log" in some places and "outputChannel" in some other files to print output. I think this should be somehow addressed consistently.
Perhaps there should be a separate issue / PR to clean this up.
We should have some global approach to print output / display feedback to the user. Now it's a kind of messy.

Reply:
outputChannel will output to output window in vscode. Now, only messages we want user see is sending to outputChannel.

Reply:
so what's the purpose of this console.log here? how user will be notified if something fails?

Reply:
it seems that if createFileShare fails you will still attempt to createDirectory, and if this fails, createFile, and finally just silently return and probably later there will be an attempt to run unexisting playbook in cloudshell?

Reply:
fixed.